### ISSUE TITLE
Transformer example crashes

### ISSUE THREAD

**Issue by FongHou** [tensorflow/swift-models#195]
Hello, I'm trying to run Transformer example, but getting this crash trace. 

```
~/tensorflow/swift-models/Transformer$ ./GPT2 0.5 "Introducing Swift for TensorFlow"
Fatal error: Input to reshape is a tensor with 768 values, but the requested shape has 7: file /swift-base/tensorflow-swift-apis/Sources/TensorFlow/Bindings/EagerExecution.swift, line 300
Current stack trace:
0    libswiftCore.so                    0x00007f5f0d297780 swift_reportError + 50
1    libswiftCore.so                    0x00007f5f0d306780 _swift_stdlib_reportFatalErrorInFile + 115
2    libswiftCore.so                    0x00007f5f0d2301d8 <unavailable> + 3793368
3    libswiftCore.so                    0x00007f5f0d230367 <unavailable> + 3793767
4    libswiftCore.so                    0x00007f5f0cff1c78 <unavailable> + 1440888
5    libswiftCore.so                    0x00007f5f0d20069e <unavailable> + 3597982
6    libswiftCore.so                    0x00007f5f0cff1399 <unavailable> + 1438617
7    libswiftTensorFlow.so              0x00007f5f0d6e7f20 dumpTensorContent<A>(_:_:) + 0
8    libswiftTensorFlow.so              0x00007f5f0d540f30 checkOk(_:file:line:) + 434
9    libswiftTensorFlow.so              0x00007f5f0d5480a0 TFE_Op.evaluateUnsafe() + 506
10   libswiftTensorFlow.so              0x00007f5f0d548790 TFE_Op.execute<A>(_:) + 132
11   libswiftTensorFlow.so              0x00007f5f0d551434 <unavailable> + 1279028
12   libswiftTensorFlow.so              0x00007f5f0d639e50 static Raw.reshape<A, B>(_:shape:) + 1057
13   libswiftTensorFlow.so              0x00007f5f0d70a3f0 Tensor.reshaped(toShape:) + 114
14   libswiftTensorFlow.so              0x00007f5f0d706420 Tensor.reshaped(to:) + 113
15   libswiftTensorFlow.so              0x00007f5f0d753c10 LayerNorm.callAsFunction(_:) + 744
16   GPT2                               0x0000563cd54275e7 <unavailable> + 251367
17   GPT2                               0x0000563cd544e475 <unavailable> + 410741
18   GPT2                               0x0000563cd53fce38 <unavailable> + 77368
19   libc.so.6                          0x00007f5f02434ab0 __libc_start_main + 231
20   GPT2                               0x0000563cd53fb19a <unavailable> + 70042
Illegal instruction

```
I'm using this S4TF build on Ubuntu 18.10 LTS

Welcome to Swift version 5.1-dev (LLVM 200186e28b, Swift 28eb536edd).

**Comment by BradLarson** [comment]
I believe this is a known issue, related to BatchNorm and LayerNorm reshaping issues we've had for a few days. I'm tracking this down and trying to get a fix and tests in place.

**Comment by 8bitmp3** [comment]
hey @BradLarson , I think I've got a similar issue with running a LeNet on MNIST in Colab. Let us know how it goes. Thanks for all the work 👍

**Comment by BradLarson** [comment]
This should be fixed, as of the nightly builds starting the August 16.
